src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -77,26 +77,27 @@ export const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(16.94),
-    color: colors.holytosca
+    color: COLORS.anon_primary
   },
   dateCreate: {
     marginLeft: 20,
     fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[400],
     lineHeight: normalizeFontSize(16.94),
-    color: '#000',
+    color: COLORS.black,
     marginTop: 4,
     marginBottom: 9
   },
   groupName: {
     fontSize: normalizeFontSize(24),
     lineHeight: normalizeFontSize(29.05),
-    color: '#000',
+    color: COLORS.black,
+    width: '100%',
     paddingHorizontal: dimen.normalizeDimen(20),
     fontWeight: 'bold'
   },
   lineTop: {
-    backgroundColor: colors.alto,
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, this change seems fine as long as the new color values in `COLORS` match the intended design. Make sure to test these changes thoroughly.
```diff
- color: colors.holytosca
+ color: COLORS.anon_primary
...
- color: '#000',
+ color: COLORS.black,
...
- color: '#000',
+ color: COLORS.black,
...
- backgroundColor: colors.alto,
+ backgroundColor: COLORS.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->